test/appium/views/base_view.py

```diff
@@ -182,6 +182,14 @@ def __init__(self, driver):
             'text': BaseText
         }
 
+    def accept_agreements(self):
```

Comment:
I'm not sure what does this method do. What kind of agreements? Can we add some detailed log or at least a doc for this method?

Reply:
We have two warning alerts when launching the app so need to close/accept them